src/scr.c

```diff
@@ -678,19 +679,37 @@ static int scr_get_params()
   /* if not enabled, bail with an error */
   if (! scr_enabled) {
     scr_param_finalize();
+    if (scr_my_rank_world == 0) {
+      // 1. Dump SCR parameters - SCR_ENABLE 
+      scr_dbg(0, "SCR_ENABLE is disabled");
+    }
     return SCR_FAILURE;
   }
-
+  else {
+    if (scr_my_rank_world == 0) {
+      // 2. Dump SCR parameters - SCR_ENABLE
+      scr_dbg(0, "SCR_ENABLE is enabled");
+    }
+  }
   /* set debug verbosity level */
   if ((value = scr_param_get("SCR_DEBUG")) != NULL) {
     scr_debug = atoi(value);
+    if (scr_my_rank_world == 0) {
+      // 3. Dump SCR parameters - SCR_aDEBUG
+      scr_dbg(scr_debug, "SCR_DEBUG level = %d", scr_debug);
+    }
   }
```

Comment:
```suggestion
  /* set debug verbosity level */
  if ((value = scr_param_get("SCR_DEBUG")) != NULL) {
    scr_debug = atoi(value);
  }
  if (scr_my_rank_world == 0) {
    // 3. Dump SCR parameters - SCR_aDEBUG
    scr_dbg(scr_debug, "SCR_DEBUG level = %d", scr_debug);
  }
```

Reply:
Let's pull the print messages outside of the ``if`` statements here.  That way it will always print the value.

And let's format things to be simple ``KEY=VALUE`` form when possible.  I think that will come in handy since it may allow us to cut-and-paste a print out like this to be included in an SCR config file for reproducing user problems.